### PR TITLE
Update in-progress extensions list.

### PR DESCRIPTION
**Pull request by donmccurdy**
Note that I'm marking `KHR_blend` and `KHR_techniques_webgl` as "tabled", to indicate they aren't current priorities but could be picked up again at some point in the future. Other suggestions welcome!

/cc @lexaknyazev @bghgary

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -25,16 +25,25 @@
...
 but may still change before they are complete._
 
 | Extension | Status |
 |-----------|--------|
-| [KHR_techniques_webgl](2.0/Khronos/KHR_techniques_webgl/README.md) | Feature-complete, seeking feedback. |
-| KHR_compressed_texture_transmission | In development. |
-| [KHR_blend](https://github.com/KhronosGroup/glTF/pull/1302) | In development. |
-| HDR textures ([#1220](https://github.com/KhronosGroup/glTF/issues/1220), [#1365](https://github.com/KhronosGroup/glTF/issues/1365)) | Planned. |
-| Advanced PBR materials ([#1221](https://github.com/KhronosGroup/glTF/issues/1221), [#1079](https://github.com/KhronosGroup/glTF/issues/1079), ...) | Planned. |
+| [KHR_image_ktx2](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation  and feedback. |
```
Comment (bghgary) [inline extensions/README.md]: nit:
```suggestion
| [KHR_image_ktx2](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation and feedback. |
```

File: extensions/README.md [hunk 2]
```diff
@@ -25,16 +25,25 @@
...
+| [KHR_texture_basisu](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation  and feedback. |
```
Comment (bghgary) [inline extensions/README.md]: nit: 
```suggestion
| [KHR_texture_basisu](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation and feedback. |
```

File: extensions/README.md (hunk 1 above)
Comment (bghgary) [reply extensions/README.md]: I think we talked about removing this extension completely, so I'm not sure I would call this ready for implementation.

File: extensions/README.md [hunk 3]
```diff
@@ -25,16 +25,25 @@
...
+| [KHR_quantized_geometry](https://github.com/KhronosGroup/glTF/pull/1673) | Ready for implementation and feedback. |
```
Comment (zeux) [inline extensions/README.md]: KHR_mesh_quantization ;-)

File: extensions/README.md (hunk 1 above)
Comment (donmccurdy) [reply extensions/README.md]: Oh! I seem to have missed that, and it's still marked as a dependency of other extensions in https://github.com/KhronosGroup/glTF/pull/1612. What's the alternative?

File: extensions/README.md (hunk 3 above)
Comment (donmccurdy) [reply extensions/README.md]: 👍

File: extensions/README.md (hunk 1 above)
Comment (lexaknyazev) [reply extensions/README.md]: That PR hasn't been changed since we talked about the updates.

Comment (donmccurdy) [reply extensions/README.md]: Understood, but I missed that and don't know what's changed. Are you suggesting I just remove this line? Or should anything else related to #1612 be changed here? Will each extension independently permit `.ktx2` images, without `KHR_image_ktx2`?

Comment (bghgary) [reply extensions/README.md]: I would just change the status to In Development instead of Ready for implementation...

Comment (donmccurdy) [reply extensions/README.md]: Done. 👍

**Pull request merged**